@xen-orchestra/xapi/host.mjs

```diff
@@ -1,11 +1,53 @@
+import TTLCache from '@isaacs/ttlcache'
 import { asyncEach } from '@vates/async-each'
 import { asyncMap } from '@xen-orchestra/async-map'
 import { decorateClass } from '@vates/decorate-with'
 import { defer } from 'golike-defer'
 import { incorrectState, operationFailed } from 'xo-common/api-errors.js'
-
 import { getCurrentVmUuid } from './_XenStore.mjs'
 
+const IPMI_CACHE_TTL = 6e4
+
+const IPMI_SENSOR_DATA_TYPE = {
```

Comment:
What the point of this dictionary which has the same keys and values?

Reply:
It's mainly due to personal preference.

I found this more readable than creating a `const` variable for each data type.
```js
const IPMI_SENSOR_TOTAL_POWER = 'totalPower'
const IPMI_SENSOR_OUTLET_TEMP =  'outletTemp'
...
```
Also, to get all `IPMI sensor data type`, we simply need to call `Object.keys()`. If we were creating a `const` for each data type, it would  require creating an array and adding each values to array like:
```js
const allIpmiDataType = [IPMI_SENSOR_TOTAL_POWER, IPMI_SENSOR_OUTLET_TEMP, ...]
```

And in case we need to add a data type, we would need to create the associated `const` and add it to the array.
By working with dictionary, we only need to add the new data type in the dictionary.